perfact/zodbsync/subcommand.py

```diff
@@ -66,6 +66,10 @@ def gitcmd_run(self, *args):
         '''Wrapper to run a git command.'''
         subprocess.check_call(self.gitcmd(*args))
 
+    def gitcmd_try(self, *args):
+        '''Wrapper to run a git command.'''
```

Comment:
Well since this is not a command that is to be run on a customer's system (taking advantage of an LTS release cycle), but on the laptop of the person that does the port, I would be willing to require at least Ubuntu 20, which has git 2.25.
Otherwise I would need to reimplement the functionality, checking for each file if it was deleted/modified/created by the commit (though I will probably not find time to do that in the next days or weeks...).

This functionality is, after all, only an intermediate tool helping the transition where some systems changed to the new formatting and some have not (or rather, some development was started when the system used the old formatting and needs to be transferred to a system that now uses the new formatting).

As a compromise, we could check for the git version and fall back to cherry-picking, but this will also not work in a lot of cases on old git versions (in particular, any commits containing the deletion of a meta file that did not end with a newline).

Reply:
Alternatively, we could start a shell and allow the user to fix merge conflicts by hand...

Reply:
I was just thinking simply executing `git rm`, followed by `git checkout` without `--no-overlay`. But this is not the same. If an object was changed, but some of its subobjects were not touched by the commit, this would remove them.

Reply:
So the question is: How many people at PerFact still use Ubuntu 18 on their work laptop?

Reply:
Well if we only use this command on work laptops i am totally fine. It was a complete coincidence that i tested on Ubuntu18 ( testing from WSL Ubuntu18 on Windows 10 😄  ), it was just the first machine available with `zodbsync` dev environment on my home pc.

Reply:
Everything works perfectly fine on Ubuntu20, just switched my dev environment